### DISCUSSION TITLE
[1062810] Inkbound [仅本体]

### DISCUSSION THREAD

**Discussion by YYMdididiidi** [pjy612/SteamManifestCache#72] [Share]
`appid:1062810
keys:
1062811;e49351772dca11a35a40ecc97cfa7253bbf0ecd03f78e78275a79b7776110da9`
[1062810.zip](https://github.com/pjy612/SteamManifestCache/files/13695086/1062810.zip)
内测code：A24EAA5139DA8304

**Comment by pjy612** [comment]
查收邮件
